Lib/typing.py

```diff
@@ -2631,7 +2630,7 @@ def __index__(self) -> int:
 
 
 @runtime_checkable
-class SupportsAbs(Protocol[T_co]):
+class SupportsAbs[T_co](Protocol):
```

Comment:
Maybe just use `T` instead of continuing to use Hungarian notation to indicate variance?

Reply:
I quite like the Hungarian notation here, actually; I like the reminder that this type variable will be inferred to be covariant. But no strong opinion.